app/src/routes/editor/WebviewArea/BrowserControl.tsx

```diff
@@ -166,4 +178,4 @@ function BrowserControls({
     );
 }
 
-export default BrowserControls;
+export default observer(BrowserControls);
```

Comment:
Not sure if this works the same way. Generally wrapped around the definition of the function like so 

```
const BrowserControls = observer(({
    webviewRef,
    webviewSrc,
    setWebviewSrc,
    selected,
    hovered,
    setHovered,
    onlookEnabled,
}: BrowserControlsProps) => {
```

Reply:
Example usage: https://github.com/kartik-raj7/onlook/blob/3767ab8ce4234a70d20f7ab620b1abb5c19586ef/app/src/components/AppBar/index.tsx#L9